client/requirements-optional.txt

```diff
@@ -0,0 +1 @@
+ray[default,data]>=2.30, <3
```

Comment:
As we commented internally Elena I was looking for references to `ray[data]` and I don't think we are using it anymore. I think it's safe to remove this optional package and we simplify the README also.